src/api/app/helpers/webui/projects/attributes_helper.rb

```diff
@@ -0,0 +1,16 @@
+module Webui::Projects::AttributesHelper
```

Comment:
This might be implemented with attributes, but this is the `Webui::Projects::CategoryHelper` IMHO

Reply:
True